docs/guides/setup/installation/components_local/_create_a_virtual_environment.mdx

```diff
@@ -0,0 +1,27 @@
+<!--
+-->
+
+Once you have confirmed that Python 3 is installed locally, you can create a virtual environment with venv.
+
+<details>
+<summary>Python Virtual Environments</summary>
+We have chosen to use venv for virtual environments in this guide, because it is included with Python 3. You are not limited to using venv, and can just as easily install Great Expectations into virtual environments with tools such as virtualenv, pyenv, etc.
+</details>
+
+Depending on whether you found that you needed to run `python` or `python3` in the previous step, you will run either:
+
+```console title="Terminal command"
+python -m venv venv
+```
+
+or
+
+```console title="Terminal command"
+python3 -m venv venv
+```
+
+This command will create a new directory called `venv` where your virtual environment is located. In order to activate the virtual environment run:
```

Comment:
Since this seems aimed at super beginners, it may be more clear to use a venv name other than venv (so they know the difference between the 2 venv in the command)

Reply:
I've pushed the corresponding update.